src/capi/azext_capi/custom.py

```diff
@@ -494,8 +494,22 @@ def check_prereqs(cmd, install=False):
 
     # Check for required environment variables
     # TODO: remove this when AAD Pod Identity becomes the default
-    for var in ["AZURE_CLIENT_ID", "AZURE_CLIENT_SECRET", "AZURE_SUBSCRIPTION_ID", "AZURE_TENANT_ID"]:
-        check_environment_var(var)
+    check_enviroment_variables()
+
+
+def check_enviroment_variables():
+    required_env_vars = ["AZURE_CLIENT_ID", "AZURE_CLIENT_SECRET", "AZURE_SUBSCRIPTION_ID", "AZURE_TENANT_ID"]
+    missing_env_vars = []
+    for var in required_env_vars:
+        if not check_environment_var(var):
+            missing_env_vars.append(var)
```

Comment:
The previous four lines could be replaced by a [list comprehension](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions):
```python
missing_env_vars = [v for v in required_env_vars if not check_environment_var(v)]
```

Reply:
Cool, good to know some python magic :)